### ISSUE TITLE
Convert screenshots to actual text in the Configuring an IDE page

### ISSUE THREAD

**Issue by Jummit** [godotengine/godot-docs#3485]
**Your Godot version:**

3.2.1

**Issue description:**

The guide expects you to copy the code seen in the screenshot, which is just unneeded work. I suggest it be converted to a code block with actual text so one can just copy it.

**URL to the documentation page (if already existing):**

https://docs.godotengine.org/en/latest/development/cpp/configuring_an_ide.html#doc-configuring-an-ide-vscode

**Comment by pycbouh** [comment]
I agree, and have to say that in its current form this article is unreadable to me. All the images are of different styles and environments and often take much more place than the text that instructs the reader. It's very hard to follow.

While some images are worth keeping in their current state, some are needed to be converted into written instructions (with a possible link or a smaller preview of the related UI), and some may even need to be removed.

According to [this](https://docutils.sourceforge.io/docs/ref/rst/directives.html#image) images can be made smaller and be aligned to a side, for example. A `target` option can be then used to link to an expanded image with full details visible, if required.

**Comment by Calinou** [comment]
Given how many images the page has, I think we should split each IDE into its own page.
